src/common/minting/commands/common/store-ipfs-object-command.spec.ts

```diff
@@ -38,31 +38,21 @@ describe('StoreIpfsObjectCommand', () => {
       it('should assign a RequiredBlobFile to state', async () => {
         const state = {} as MintState;
 
-        const expected = { error: MintError.RequiredBlobFile };
+        const expected = { ...state, error: MintError.RequiredBlobFile };
```

Comment:
Why are you using copying an empty object inside another object?

Reply:
bcuz that's how the code being tested is behaving, not a particular reason tbh